docs/source/contents/4_draft_mapping.rst

```diff
@@ -668,6 +670,195 @@ All values within <tableOfContents> will be mapped to RDF in the same way. Below
+
+.. code-block:: turtle
+    :caption: Resulting RDF `from harp:1 <https://digital.lib.utk.edu/collections/islandora/object/harp%3A1/datastream/MODS>`_
+    :name: Resulting RDF `from harp:1 MODS <https://digital.lib.utk.edu/collections/islandora/object/harp%3A1/datastream/MODS>`_
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+
+    <https://example.org/objects/1>
+        relators:cmp <http://id.loc.gov/authorities/names/no2002022963> .
+
+When there is no :code:`mods:name/@valueURI`, use the string literal from :code:`mods:name/mods:namePart`:
+
+.. code-block:: xml
+    :caption: XML with Name missing a valueURI
+    :name: XML with Name missing a valueURI
+
+    <name type="personal">
+        <namePart>Daniel, Charles R. (Charlie), Jr., 1930-</namePart>
+        <role>
+            <roleTerm type="text" authority="marcrelator" valueURI=" http://id.loc.gov/vocabulary/relators/cre">Creator</roleTerm>
+        </role>
+    </name>
+
+.. code-block:: turtle
+    :caption: Resulting turtle for Name missing a valueURI
+    :name: Resulting turtle for Name missing a valueURI
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+
+    <https://example.org/objects/1>
+        relators:cre "Daniel, Charles R. (Charlie), Jr., 1930-" .
+
+If there is a :code:`mods:name/valueURI` but it's empty, use the string literal instead:
+
+.. code-block:: xml
+    :caption: Example XML from `volvoices:2495 MODS <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2495/datastream/MODS>`_
+    :name: Example XML from `volvoices:2495 MODS <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2495/datastream/MODS>`_
+
+    <name authority="naf" type="corporate" valueURI="">
+        <namePart>Bemis Bro. Bag Company</namePart>
+        <role>
+            <roleTerm authority="marcrelator" type="text" valueURI="http://id.loc.gov/vocabulary/relators/asn">Associated name</roleTerm>
+        </role>
+    </name>
+
+.. code-block:: turtle
+    :caption: Resulting turtle from `volvoices:2495 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2495/datastream/MODS>`_
+    :name: Resulting turtle from `volvoices:2495 <https://digital.lib.utk.edu/collections/islandora/object/volvoices:2495/datastream/MODS>`_
+
+    @prefix relators: <http://id.loc.gov/vocabulary/relators/> .
+
+    <https://example.org/objects/1>
+        relators:asn "Bemis Bro. Bag Company" .
+
+Keep all Roles
+--------------
+
+Use Case
+^^^^^^^^
+
+Occassionally, a :code:`mods:name` will have multiple roles.  When this happens, keep them all.
+
+Justification
+^^^^^^^^^^^^^
+
+It's important that we keep the relationship between people and our digital object.
+
+Xpaths
+^^^^^^
+:code:`count(mods:name/mods:role)>1`
+
+Decision
+^^^^^^^^
+
+.. code-block:: xml
+    :caption: `harp:1 XML record <https://digital.lib.utk.edu/collections/islandora/object/harp%3A1/datastream/MODS>`_ has multiple role terms
```

Comment:
In the previous example for roleTerms you used this same record. Since this record has more than one roleTerm, I don't think it's appropriate to use in both the previous example and this example. Can you pick another record that has just one roleTerm for it? We also might want to slightly rename the use case so that it's clear that we're dealing with names that have a single roleTerm. "Leverage Marc Relators for RDF Property Value and Relationship to the Digital Object" doesn't tell us anything about the number. In a way this could be included as an additional example in the previous bucket if you'd rather do that as well.